es-ds-docs/pages/molecules/verification-code.vue

```diff
@@ -0,0 +1,124 @@
+<script setup lang="ts">
+const { $prism } = useNuxtApp();
+const compCode = ref('');
+const docCode = ref('');
+
+if ($prism) {
+    /* eslint-disable import/no-webpack-loader-syntax, import/no-self-import */
+    const compSource = await import('@energysage/es-ds-components/components/es-verification-code.vue?raw');
+    const docSource = await import('./verification-code.vue?raw');
+    /* eslint-enable import/no-webpack-loader-syntax, import/no-self-import */
+
+    compCode.value = $prism.normalizeCode(compSource.default);
+    docCode.value = $prism.normalizeCode(docSource.default);
+    $prism.highlight();
+}
+
+const propTableRows = [
+    ['v-model', 'String', 'n/a', 'Required. The v-model directive binds the otpinput to a data property.'],
+];
+const code = ref();
+const isSubmitInProgress = ref(false);
+
+const charCount: number = 5;
+
+const isValid = () => {
+    if (code.value && code.value.length === charCount) {
+        return true;
+    }
+    return false;
+};
+
+const randomCode = () => {
+    // Randomly generates an array of numbers 0-9
+    const rand = Array.from({ length: charCount }, () => Math.floor(Math.random() * 10).toString());
+
+    // eslint-disable-next-line no-console
+    console.log('Generated', rand);
+    isSubmitInProgress.value = true;
+    setTimeout(() => {
+        isSubmitInProgress.value = false;
+    }, 1000);
+    return rand.join('');
+};
+
+// eslint-disable-next-line no-return-assign
+const clearCode = () => {
+    isSubmitInProgress.value = true;
+    setTimeout(() => {
+        isSubmitInProgress.value = false;
+    }, 1000);
+    return '';
+};
+</script>
+
+<template>
+    <div>
+        <h1>Verification code</h1>
+        <p>{{ code }}</p>
```

Comment:
We probably don't need this, as the code is displayed in the inputs themselves, and the old docs page didn't have this.